commons/all/src/main/java/org/infinispan/commons/hash/CRC16.java

```diff
@@ -21,6 +16,7 @@
  */
 @ThreadSafe
 @Immutable
+@ProtoTypeId(ProtoStreamTypeIds.HASH_CRC16)
```

Comment:
It is not present in the schema and I'm guessing it is not sent over the wire. There is no proto factory either.